src/widgets/bookmark/BookmarkWidgetTile.tsx

```diff
@@ -226,26 +241,44 @@ function BookmarkWidgetTile({ widget }: BookmarkWidgetTileProps) {
     case 'vertical':
       return (
         <ScrollArea
-          offsetScrollbars
-          type="always"
+          scrollbarSize={8}
+          type="auto"
           h="100%"
           mr={isEditModeEnabled ? 'xl' : undefined}
+          styles={{
+            viewport:{
+              //mantine being mantine again... this might break
+              '& div[style="min-width: 100%; display: table;"]':{
```

Comment:
@Meierschlumpf I wrote this together yesterday with OP. This is the only way, right?

Reply:
I guess it's fine